wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -27,14 +28,22 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     */
   override def lock(): LockedWalletApi = {
     logger.debug(s"Locking wallet")
-    LockedWallet(nodeApi, chainQueryApi)
+    LockedWallet(nodeApi, chainQueryApi, addressTagDAOs)
   }
 
   override def sendToAddress(
       address: BitcoinAddress,
       amount: CurrencyUnit,
       feeRate: FeeUnit,
-      fromAccount: AccountDb): Future[Transaction] = {
+      fromAccount: AccountDb): Future[Transaction] =
+    sendToAddress(address, amount, feeRate, fromAccount, Vector.empty)
+
+  def sendToAddress(
+      address: BitcoinAddress,
+      amount: CurrencyUnit,
+      feeRate: FeeUnit,
+      fromAccount: AccountDb,
+      newTags: Vector[AddressTag]): Future[Transaction] = {
```

Comment:
Why are `newTags` used here?

Reply:
Adds an argument to sendToAddress that is newTags: Vector[AddressTag] that will replace the current tags of the spent utxos to the new tags for the subsequent change utxo